SPECS/nginx/nginx.spec

```diff
@@ -3,8 +3,8 @@
 
 Summary:        High-performance HTTP server and reverse proxy
 Name:           nginx
-Version:        1.20.1
```

Comment:
This is also a big jump.  But probably ok.  Are there any other Mariner packages that depend on this one?